ESP8266Interface.h

```diff
@@ -40,7 +40,7 @@ class ESP8266Interface : public NetworkStack, public WiFiInterface
      * @param rx        RX pin
      * @param debug     Enable debugging
      */
-    ESP8266Interface(PinName tx, PinName rx, bool debug=false, PinName rts=NC, PinName cts=NC);
+    ESP8266Interface(PinName tx, PinName rx, bool debug=false, PinName rts=MBED_CONF_ESP8266_RTS, PinName cts=MBED_CONF_ESP8266_CTS);
```

Comment:
Does this still compile, if you are not using this board at all and those have not been defined?
That's the requirement this driver needs to pass.